.github/workflows/ci.yml

```diff
@@ -14,3 +14,37 @@ jobs:
     # Only run cron on the silverstripe account
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
     uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    with:
+      # set phpunit to false to prevent automatic generation of mysql phpunit jobs
+      phpunit: false
+      extra_jobs: |
+        - php: 8.1
+          db: sqlite3
+          phpunit: true
+          composer_args: --prefer-lowest --prefer-source
```

Comment:
We don't need `--prefer-source` for any of these - we'll add a new input `preserve_vendor_tests` in https://github.com/silverstripe/gha-ci/pull/58/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR42

Reply:
DONE